src/components/PageApplicationComments/PageApplicationComments.tsx

```diff
@@ -59,6 +65,10 @@ export const PageApplicationComments = ({
   searchParams,
   comments,
 }: PageApplicationCommentsProps) => {
+  const commentSearchFields = useMemo(
+    () => getEnabledFields(COMMENT_SEARCH_FIELDS, disabledCommentSearchFields),
+    [],
+  );
```

Comment:
ah interesting, we're doing a lot here and we probably wouldn't need useMemo here if we did the calculation inside featureFlag.ts! 🤔